nursery/qt/toga_qt/libs/utils.py

```diff
@@ -0,0 +1,19 @@
+try:  # try import PyQt5
+    from PySide2.QtCore import qVersion
+    GUI = 'pyside2'
+
+except ImportError:
+    try:  # try import PySide2
+        from PySide.QtCore import qVersion
```

Comment:
Again - this isn't needed. We can *enforce* the availability of PySide2. There no need for a fallback here.

Reply:
dont mind just added for pyside1 users

Reply:
@freakboy3742  is it ok

Reply:
I don't know how many times I can say it. We can (and will) *enforce* the presence of PySide2. There will be *exactly zero* users of Toga-qt who do not have PySide2 installed. There is *absolutely no use case* for a fallback when you can *guarantee* the presence of your preferred API.

Reply:
ok